clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentWgsCram.scala

```diff
@@ -1,62 +1,64 @@
 package org.broadinstitute.clio.server.dataaccess.elasticsearch
 
+import java.net.URI
 import java.time.OffsetDateTime
+import java.util.UUID
 
 import org.broadinstitute.clio.util.model.{DocumentStatus, Location, UpsertId}
 
 case class DocumentWgsCram(
   upsertId: UpsertId,
-  entityId: String,
+  entityId: Symbol,
   location: Location,
   project: String,
   sampleAlias: String,
   version: Int,
   documentStatus: Option[DocumentStatus] = None,
-  pipelineVersion: Option[String] = None,
+  pipelineVersion: Option[Symbol] = None,
   workflowStartDate: Option[OffsetDateTime] = None,
   workflowEndDate: Option[OffsetDateTime] = None,
-  cramMd5: Option[String] = None,
+  cramMd5: Option[Symbol] = None,
   cramSize: Option[Long] = None,
-  cramPath: Option[String] = None,
-  craiPath: Option[String] = None,
-  cramMd5Path: Option[String] = None,
-  logPath: Option[String] = None,
-  fingerprintPath: Option[String] = None,
-  cromwellId: Option[String] = None,
-  workflowJsonPath: Option[String] = None,
-  optionsJsonPath: Option[String] = None,
-  wdlPath: Option[String] = None,
-  readgroupMd5: Option[String] = None,
+  cramPath: Option[URI] = None,
+  craiPath: Option[URI] = None,
+  cramMd5Path: Option[URI] = None,
+  logPath: Option[URI] = None,
+  fingerprintPath: Option[URI] = None,
+  cromwellId: Option[UUID] = None,
```

Comment:
I chose `UUID` for representing cromwell IDs because that's what the cromwell API client uses: https://github.com/broadinstitute/cromwell/blob/29_hotfix/cromwellApiClient/src/main/scala/cromwell/api/model/WorkflowId.scala#L14